docs/DeveloperGuide.md

```diff
@@ -154,6 +154,40 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### List Meeting feature
+
+Syntax: `listMeeting`
+
+Purpose: View all `Meeting` from the Meeting List in `Model`.
+
+#### Implementation
+
+Usage Scenario of `listMeeting`:
+
+1) User inputs `listMeeting` to view the current meetings in the `Model`.
+
+Below is a sequence diagram that illustrates the execution of `listMeeting` command and the interaction with `Model`.
+
+![ListMeetingSequenceDiagram](images/ListMeetingSequenceDiagram.png)
+
+### Different view panels
+
+The GUI changes view panels depending on the last executed command. For example, a `listMeeting` will cause the meeting list view panel to be displayed, while `viewClient i/1` will cause a detailed client view panel to be displayed.
+
+#### Implementation
+
+Below is a sequence diagram that illustrates the execution of `listMeeting` command and the interaction with `Model`.
+
+![DifferentViewPanelsSequenceDiagram](images/DifferentViewPanelsSequenceDiagram.png)
```

Comment:
Was thinking if the model is necessary in this context.

Reply:
Yea was considering it as well, though I think I might have to consult TA for this. But thanks for pointing out!

Reply:
Will message him and change accordingly to his comments